Clients/GUI/wwwroot/css/site.css

```diff
@@ -181,3 +181,65 @@ app {
         display: block;
     }
 }
+
+.newsbar {
```

Comment:
Are u really sure we need custom styles?

Reply:
I really dont have time learn bootstrap

Reply:
Strange that you have time to learn pure css from a scratch, but no time for like 20 most common bootstrap ready classes, but ok